discovery/generator.go

```diff
@@ -0,0 +1,52 @@
+package discovery
+
+import (
+	"fmt"
+	"regexp"
+)
+
+func incString(s string) string {
```

Comment:
Also, if this is incrementing the IP addresses, why not parsing that as int's and simply incrementing them, instead of doing this? It seems like it would be much simpler